l10n/Makefile

```
@@ -0,0 +1,54 @@
+PHONY += pot merge check
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)